lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -21,14 +21,10 @@ define('NORTHSTAR_URL', variable_get('dosomething_northstar_url', 'http://norths
  */
 function dosomething_northstar_client() {
   return new Northstar([
-    'grant' => 'client_credentials',
+    'grant' => 'authorization_code',
     'url' => NORTHSTAR_URL,
     'bridge' => PhoenixOAuthBridge::class,
-    'client_credentials' => [
-      'client_id' => variable_get('dosomething_northstar_app_id', 'trusted-test-client'),
-      'client_secret' => variable_get('dosomething_northstar_app_key', 'secret1'),
-      'scope' => ['admin', 'user'],
-    ],
+    // @TODO: Does other stuff need to go here?
```

Comment:
The client ID and secret for the auth code grant should be set here:

```php
    'authorization_code' => [
        'client_id' => 'example',
        'client_secret' => 'xxxxxxxxxxxxx',
        'scope' => ['user'],
        'redirect_uri' => 'user/login',
    ],
```

You should be able to grab the client ID & secret from the [`openid_connect` module's variables](https://github.com/sanduhrs/drupal-openid_connect/blob/82ba5ed175a6cc8ae72124112c975a8b35c2e151/openid_connect.module#L98).